format.sh

```diff
@@ -286,6 +286,29 @@ else
 fi
 echo 'vLLM clang-format: Done'
 
+echo 'vLLM actionlint:'
+
+DOCKER=""
+if command -v "docker" &>/dev/null; then
+    DOCKER="docker"
+elif command -v "podman" &>/dev/null; then
+    DOCKER="podman"
+else
+    echo "Docker or Podman are not installed. Please install one if you want to lint GitHub CI configuration."
+fi
+
+actionlint() {
+    if [ -z "$DOCKER" ]; then
+        return
+    fi
+    # Ensure we use the same version of actionlint as CI
+    IMAGE="vllm/actionlint:latest"
+    ${DOCKER} build --tag "${IMAGE}" - < .github/workflows/actionlint.dockerfile
```

Comment:
Can we use something like `flake8` instead of Docker for local formatting? Some people might not have Docker installed ...

Reply:
It will at least skip this step gracefully if it's not installed.

The alternative is that `go` needs to be installed. It's not a Python tool. https://github.com/rhysd/actionlint

Reply:
They have pre-built binaries, so I could have it auto-download that. I'll look at that now.

Reply:
@khluu I've got it using a prebuilt binary from a github release if it's not found, so there are no longer any dependencies for this to work

Reply:
Thanks!